lib/taurus/qt/qtgui/__init__.py

```diff
@@ -54,16 +55,34 @@
 # Note: this is an experimental feature introduced in v 4.3.0a
 # It may be removed or changed in future releases
 
+class LazyModule(ModuleType):
+
+    def __init__(self, name, package, entry_point):
+        super(LazyModule, self).__init__(name)
+        self.__package__ = package
+        self.ep = entry_point
+
+    def __getattr__(self, member):
+        import sys as __sys
+        mod = self.ep.load()
```

Comment:
I think that this should be protected, since it is the point at which problems typically occur (e.g. due to missing dependencies for the plugin).
In fact, I think that the try-except block of the original code makes more sense here than in its original place now that the loading is delayed

Reply:
You're right, will change